pageserver/src/config.rs

```diff
@@ -55,6 +56,8 @@ pub mod defaults {
     pub const DEFAULT_CONCURRENT_TENANT_SIZE_LOGICAL_SIZE_QUERIES: usize =
         super::ConfigurableSemaphore::DEFAULT_INITIAL.get();
 
+    pub const DEFAULT_METRIC_COLLECTION_INTERVAL: &str = "60 s";
```

Comment:
This might be a nice interval for development. Can we ensure that in real environments it will be minutes? It might be enough for production to send each 10-15 minutes.

Reply:
this is a pageserver's config parameter `metric_collection_interval`, so we can set it to any value for production.

but why 10-15 minutes? If we're going to use this info for billing, we must be more precise.

Reply:
Usually, billing systems provide slices on an hourly basis. Therefore, @antonyc suggestion sounds reasonable to me, at least.

Reply:
> but why 10-15 minutes? If we're going to use this info for billing, we must be more precise.

The Pageserver metrics are billed as max GB consumed per hour by a timeline/tenant, so having 6 events per hour per timeline/tenant might be enough to provide reasonable reliability (it's hard to loose all the 6 events), precision, and amount of events.